lib/internal/bootstrap_node.js

```diff
@@ -76,6 +71,12 @@
       });
 
     } else if (process.argv[1] === 'inspect' || process.argv[1] === 'debug') {
+      if (process.argv[1] === 'debug') {
+        process.emitWarning(
+            '`node debug` is deprecated. Please use `node inspect` instead.',
```

Comment:
This assumes that this PR will land after #12197. @jasnell I am changing the message for `DEP0062`, is that acceptable, or should this be a different deprecation ID? Note that the behaviour is unchanged `node debug` used to print `DEP0062` and it will continue to do so, but with a better message.